packages/now-build-utils/src/index.ts

```diff
@@ -64,12 +64,11 @@ export {
 };
 
 export {
-  detectRoutes,
```

Comment:
I think this will have to be a major version bump

Reply:
@styfle yes